### PR TITLE
Conv transpose eltwiseadd bn fuse pass

### PR DESCRIPTION
**Pull request by fengxiaoshuai**
### PR types
Others

### PR changes
Others

### Describe
Conv transpose eltwiseadd bn fuse pass

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_eltwiseadd_bn_fuse_pass.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+            max_duration=250,
+            passes=["conv_transpose_eltwiseadd_bn_fuse_pass"])
+
+    def sample_program_config(self, draw):
+        # generate random number
+        random_batch_size = draw(st.integers(min_value=1, max_value=4))
+        random_channel = draw(st.integers(min_value=2, max_value=64))
+        random_input_dim1 = draw(st.integers(min_value=50, max_value=512))
+        random_input_dim2 = draw(st.integers(min_value=50, max_value=512))
+        random_groups = draw(st.sampled_from([1]))
```
Comment (jiweibo) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_eltwiseadd_bn_fuse_pass.py]: groups参数sample过少

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_eltwiseadd_bn_fuse_pass.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        random_dilations = draw(
+            st.lists(
+                st.integers(
+                    min_value=1, max_value=1), min_size=2, max_size=2))
```
Comment (jiweibo) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_eltwiseadd_bn_fuse_pass.py]: 同groups

File: python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_eltwiseadd_bn_fuse_pass.py [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+            if program_config.ops[0].attrs['groups'] != 1:
+                return True
+            return False
+
+        self.add_ignore_check_case(
+            teller1, IgnoreReasons.PASS_ACCURACY_ERROR,
+            "The output format of conv2d_transpose is wrong when data_format attribute is NHWC"
+        )
+        self.add_ignore_check_case(teller2, IgnoreReasons.PASS_ACCURACY_ERROR,
+                                   "there is diff when group >1 in this pass")
```
Comment (jiweibo) [inline python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_eltwiseadd_bn_fuse_pass.py:243]: PassCompat加下限制吧

Comment (fengxiaoshuai) [reply python/paddle/fluid/tests/unittests/ir/inference/test_conv_transpose_eltwiseadd_bn_fuse_pass.py:243]: > PassCompat加下限制吧

好的，已经添加

**Review by jiweibo — APPROVED** [review]
LGTM

**Review by XieYunshen — APPROVED** [review]
LGTM for `set_tests_properties(test_conv_transpose_eltwiseadd_bn_fuse_pass PROPERTIES TIMEOUT 250)`

**Pull request merged**